tests/qchem/hf_tests/test_tapering.py

```diff
@@ -589,17 +589,15 @@ def test_transform_hf(generators, paulixops, paulix_sector, num_electrons, num_w
         ),
     ],
 )
-def test_hf_energy(symbols, geometry, charge):
-    r"""Test that HF energy obtained from the tapered Hamiltonian and tapered Hartree Fock state
-    is consistent."""
+def test_taper_obs(symbols, geometry, charge):
+    r"""Test that values obtained from the tapered observables and tapered Hartree Fock state
```

Comment:
Could you please make this sentence more expressive? 
For instance, you might define what are the "values" (expectation values, right?) and mention that they are obtained as <hf|obs|hf>.